src/models/User.ts

```diff
@@ -0,0 +1,67 @@
+import firebase from 'firebase';
+import {
+  Grade, ScholarshipType, Gender, ScholarshipAmount, States,
+} from './enums';
+
+export enum Role{
```

Comment:
I'm surprised the linter doesn't complain about the missing space before the curly brace. Does `yarn lint --fix` do this?
```typescript
export enum Role {
```

Reply:
Usually `lint --fix` adds the space. I forgot to run the command so most of my code will have lint errors, I'll fix them with the next push.